src/components/books/BookCover.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="w-full md:w-1/3">
+    <div
+      class="book-cover image max-w-[350px]"
+      data-cy="book-cover"
+    >
+      <a
+        v-lazy-container="{ selector: 'img', error: defaultImg, loading: defaultImg }"
+        :href="item.url"
+        target="_blank"
+        rel="noopener"
+        data-cy="book-cover"
+        @click="$emit('book-cover-click')"
```

Comment:
This should be `book-title-click` since it's the one being used elsewhere and it's declared in the `emits` list.